sound/soc/fsl/fsl_dai.c

```diff
@@ -0,0 +1,276 @@
+// SPDX-License-Identifier: GPL-2.0
+//
+// Freescale Generic DAI driver for DSP
+//
+// Copyright 2019 NXP
```

Comment:
2020, Happy New year!

Reply:
Oh, will change this. I started coding this in October. But only now got the time to clean it up.